src/App.tsx

```diff
@@ -1,61 +1,145 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+
+type Togo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
```

Comment:
The type `Togo` seems to be a typo or misnaming. It should likely be `Todo` to match the context of a todo application.